fix/gsi/convinfo.rtma

```diff
@@ -26,7 +26,7 @@
  ps       182    0    1     1.5      0      0      0   5.0   3.0   1.0  10.0  0.000	   0    0.     0.      0    0.     0.
  ps       183    0   -1     0.25     0      0      0   5.0   3.0   1.0  10.0  0.000	   0    0.     0.      0    0.     0.
  ps       187    0    1     0.25     0      0      0   5.0   3.0   1.0  10.0  0.000	   0    0.     0.      0    0.     0.
- ps       188    0    1     0.1      0      0      0   5.0   3.0   1.0  10.0  0.000	   0    0.     0.      0    0.     0.
+ ps       188    0    1     0.25     0      0      0   5.0   3.0   1.0  10.0  0.000	   0    0.     0.      0    0.     0.
```

Comment:
We should also consider adding the report types 192-195 for t, ps, and q, and 292-295 for w.  These are reports that arrive with missing surface pressure.  Please see the following for more details: https://www.emc.ncep.noaa.gov/mmb/data_processing/prepbufr.doc/table_19.htm

Reply:
@guoqing-noaa Should we also consider increasing the twindow for mesonet wind obs?

Reply:
Thanks @MatthewMorris-NOAA  I forgot to change for mesonet winds. Will do soon.